src/models/sunburst.js

```diff
@@ -187,6 +196,7 @@ nv.models.sunburst = function() {
         mode:       {get: function(){return mode;}, set: function(_){mode=_;}},
         id:         {get: function(){return id;}, set: function(_){id=_;}},
         duration:   {get: function(){return duration;}, set: function(_){duration=_;}},
+        groupColorByParent: {get: function(){return groupColorByParent;}, set: function(_){groupColorByParent=_;}},
```

Comment:
please do groupColorByParent=!!_;  so that it guarantees that it saves a boolean.